.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+on: [push, pull_request]
```

Comment:
This didn't run, and I'm not immediately sure why, but I'll bet it has to do with this line.

Reply:
It is currently running on my local branch. I think I sent you links via gitter.